apps/demo/src/app/cart/components/cart-wrapper/cart-wrapper.component.html

```diff
@@ -2,22 +2,12 @@
   <div class="cart-wrapper__main">
```

Comment:
I imagine there's a lot of styles in the `cart-wrapper.component.scss` file that could be removed now. Also, the `<div class="wrapper">` could also probably be removed.